.gitattributes

```diff
@@ -41,3 +41,15 @@
 *.mo binary
 *.pdf binary
 *.phar binary
+
+# Remove files for archives generated using `git archive`
+.github export-ignore
+plugins/BEdita/API/postman export-ignore
```

Comment:
Uhm… I wasn't sure about this. Those files are useful for developers, and an archive generated using `git archive` can't be used for development. That's why I excluded them from the archive. But I wasn't frankly sure about this choice.